aiida_zeopp/parsers/plain.py

```diff
@@ -6,7 +6,7 @@
 from aiida.orm import Dict
 
 
-class KeywordParser(object):
```

Comment:
actually, you can remove the `object` in python3

Reply:
I'll remove them in the next push.